### ISSUE TITLE
Language Transfer Partition — Adapter-Seen vs. Adapter-Unseen

### ISSUE THREAD

**Issue by engichang1467** [Cohere-Labs-Community/expedition-tayavision#62]
## Context

Tiny Aya Base supports 70+ languages, but the multimodal adapter (Stage 1 + Stage 2) trains on only ~20–25 languages drawn from LLaVA-Pretrain (with ~14% multilingual supplement) and M3IT subsets. The gap between the backbone's language coverage and the adapter's training coverage is a research contribution in itself.

**Key question (Gap 11 from the updated proposal):** Does the 3.35B multilingual backbone enable zero-shot visual transfer to languages absent from multimodal training, using English as a bridge?

The partition is defined against whichever Phase 2 milestone was reached so the seen/unseen split is fixed before Δ_transfer is computed. Since multilingual training (if Milestone B/C was reached) may expand the seen set, the partition should reflect the specific training data from the best Phase 2 checkpoint (Issue #61 ).

---

### 1. Audit training data language coverage

Enumerate every language present in the multimodal training data used in Phase 2, with source attribution:

| Language | Script | Stage 1 source | Stage 2 source |
|----------|--------|---------------|----------------|
| English | Latin | LLaVA-Pretrain (primary) | LLaVA-mix |
| French | Latin | NLLB translation | M3IT / translated |
| ... | ... | ... | ... |

Document languages that are *technically seen but underrepresented* (e.g., present in ≤1% of training tokens) — flag these as "marginal seen."

### 2. Partition CVQA (31 languages)

Assign each CVQA language to one of three groups:
- **Adapter-Seen:** Substantial representation in Stage 1 or Stage 2 training data
- **Marginal Seen:** Present but at <1% of total tokens
- **Adapter-Unseen:** Supported by Tiny Aya Base but completely absent from multimodal training

Save as `docs/data/cvqa_language_partition.csv`.

### 3. Partition Kaleidoscope (18 languages)

Same three-way classification as CVQA.  
Save as `docs/data/kaleidoscope_language_partition.csv`.

### 4. Define the Δ_transfer metric

```
Δ_transfer(lang) = CVQA_accuracy(adapter-seen languages) − CVQA_accuracy(adapter-unseen languages)
```

A small Δ_transfer (< 5 pp) → strong zero-shot visual transfer from the multilingual backbone.  
A large Δ_transfer (> 15 pp) → visual grounding is bottlenecked by language-specific adapter training.

This metric is computed and reported in Phase 3 using scores from Issue 01. The Δ_transfer result is a confirmed paper deliverable (Gap 11). This issue defines the partition; Issue 01 supplies the per-language scores.

---

## Resources

| Resource | Details |
|----------|---------|
| Training data manifest | Final data mix from Issue #17 PR / W&B run config. |
| CVQA language list | Available in dataset card at `afaji/cvqa` on HF Hub. |
| Kaleidoscope language list | Available in dataset card at `CohereLabs/Kaleidoscope` on HF Hub (check exact name). |
| Tiny Aya language list | Full 70+ language list in Tiny Aya tech report (uploaded to NotebookLM as `tiny_aya_tech_report.pdf`). |

---

## Acceptance Criteria

- [ ] Training data language audit complete — every language with ≥0.1% representation documented with source attribution.
- [ ] CVQA partition table saved at `docs/data/cvqa_language_partition.csv` (columns: language, script, group, notes).
- [ ] Kaleidoscope partition table saved at `docs/data/kaleidoscope_language_partition.csv`.
- [ ] Marginal-seen cases flagged and discussed in `docs/data/partition_notes.md`.
- [ ] Partition reviewed by at least one collaborator.
- [ ] Δ_transfer formula and interpretation documented in `docs/data/partition_notes.md`.

---

## Output Artifacts

- `docs/data/training_language_audit.csv` — per-language coverage with source attribution
- `docs/data/cvqa_language_partition.csv`
- `docs/data/kaleidoscope_language_partition.csv`
- `docs/data/partition_notes.md` — Δ_transfer definition + interpretation guidance
